sql/migrations/024-alter-evaluations-table-add-column-history-id.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE evaluations
+ADD COLUMN `history_id` int DEFAULT NULL AFTER `data`;
```

Comment:
Should we add an index on this column (and on the corresponding column in the orders table)?